### PR TITLE
[bulk] validations: duplicateH1Title issues

### PR DESCRIPTION
**Pull request by Vette99**
This PR fixes 12 duplicateH1Title issues and 4 missing-description issues and 2 duplicate Description issues

### CODE REVIEW

File: Skype/SDN/articles/audiotimestamperrormicms-element-qualitypropertiestype-complextype-skype-for-business-sdn-interface-2-2-schema-c.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
 ﻿---
-title: AudioTimestampErrorMicMs element (QualityPropertiesType complexType) 
+title: AudioTimestampErrorMicMs element (QualityPropertiesType complexType) Schema C
```
Comment (reemsabaw1) [inline Skype/SDN/articles/audiotimestamperrormicms-element-qualitypropertiestype-complextype-skype-for-business-sdn-interface-2-2-schema-c.md:2]: Does Schema C have to be in parenthesis here?

File: Skype/SDN/articles/audiotimestamperrormicms-element-qualitypropertiestype-complextype-skype-sdn-2-2-c.md [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 ms:contentKeyID: 68250614
+description: Speaking device clock drift rate, relative to CPU clock. Average error of microphone-captured-stream time stamp, in milliseconds, for last 20 seconds of a call.
 ms.date: 08/24/2015
 mtps_version: v=office.16
 dev_langs:
 - xml
 ---
 
-# AudioTimestampErrorMicMs element (Schema C)
+# AudioTimestampErrorMicMs element Schema C
```
Comment (reemsabaw1) [inline Skype/SDN/articles/audiotimestamperrormicms-element-qualitypropertiestype-complextype-skype-sdn-2-2-c.md:14]: Does Schema C have to be in parenthesis here?

File: Skype/SDN/articles/audiotimestamperrorspkms-element-qualitypropertiestype-complextype-skype-sdn-2-2-d.md [hunk 3]
```diff
@@ -1,16 +1,17 @@
 ﻿---
-title: AudioTimestampErrorSpkMs element (QualityPropertiesType complexType) 
+title: AudioTimestampErrorSpkMs element (QualityPropertiesType complexType) Schema D
```
Comment (reemsabaw1) [inline Skype/SDN/articles/audiotimestamperrorspkms-element-qualitypropertiestype-complextype-skype-sdn-2-2-d.md:2]: parenthesis around Schema D?

**Comment by Vette99** [comment]
@lindalu-MSFT
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Pull request merged**